packages/sanity/src/structure/useStructureToolSetting.ts

```diff
@@ -4,23 +4,26 @@ import {useSettingsStore} from 'sanity'
 /**
  * @internal
  */
-export function useDeskToolSetting<ValueType>(
+export function useStructureToolSetting<ValueType>(
   namespace: string | null,
   key: string,
   defaultValue?: ValueType,
 ): [ValueType | undefined, (_value: ValueType) => void] {
   const settingsStore = useSettingsStore()
   const [value, setValue] = useState<ValueType | undefined>(defaultValue)
 
-  const deskToolSettings = useMemo(() => settingsStore.forNamespace('desk-tool'), [settingsStore])
+  const structureToolSettings = useMemo(
+    () => settingsStore.forNamespace('structure-tool'),
```

Comment:
I think you likely considered this (and it's probably an acceptable trade-off), but any desk tool settings currently stored in localStorage (such as sort orders) will be disregarded after this change.

Reply:
Yeah. It is a slightly "breaking" change in that settings for panes (ordering and display mode) will be lost, but I think it is acceptable given all you have to do is to set the ordering once more.